mslib/msui/mscolab.py

```diff
@@ -74,7 +76,8 @@ class MSSMscolabWindow(QtWidgets.QMainWindow, ui.Ui_MSSMscolabWindow):
     identifier = None
     viewCloses = QtCore.pyqtSignal(name="viewCloses")
 
-    def __init__(self, parent=None, data_dir=mss_default.mss_dir, mscolab_server_url=mss_default.mscolab_server_url):
+    def __init__(self, parent=None, data_dir=config_loader(dataset="mss_dir"),
```

Comment:
Does this work? Timing wise? Always? I would prefer None here with access to config_loader in the function if it is None, similar to how one should handle dicts/lists.